data-raw/otu_small.R

```diff
@@ -8,8 +8,7 @@ usethis::use_data(otu_small, overwrite = TRUE)
 ## code to prepare models with the `otu_small` otu_small
 set.seed(2019)
 outcome_colname <- "dx"
-outcome_value <- "cancer"
-kfolds <- 5
+kfolds <- 2
```

Comment:
Probably should just delete these variables since they don't actually get used.

Reply:
Deleted kfolds, but we do use outcome_colname so I kept that one. We can change more if we want when we clean up how we're organizing the data.